react/TextLink/TextLink.js

```diff
@@ -13,9 +13,9 @@ const renderChevron = chevron => {
 
   return (
     <ChevronIcon
+      standard
```

Comment:
TextLink may need to be decorated `withTextProps` and pass it through to here, as a `TextLink` relies on being either a parent/child of `Text` to set the font size, the chevron should honour this.

Reply:
Agree, that’s the right way. Was just hardcoded previously, so was not worried. I will change 👍